juniper_warp/src/lib.rs

```diff
@@ -200,12 +217,12 @@ where
 /// let graphql_filter = make_graphql_filter(schema, context_extractor);
 ///
 /// let graphql_endpoint = warp::path("graphql")
-///     .and(warp::post2())
+///     .and(warp::post())
```

Comment:
Why this change?

Reply:
I decided to use wapr's master branch because it supports futures 0.3 and they recently [replaced all `get`, `post`, `put` and `delete` functions with `get2`, `post2`, `put2` and `delete2`](https://github.com/seanmonstar/warp/commit/b9637567f96c64f1bfe734842718e28a7c633e23)